guides/source/routing.md

```diff
@@ -76,20 +91,22 @@ Rails.application.routes.draw do
 end
 ```
 
-Since this is a regular Ruby source file you can use all of its features to help you define your routes but be careful with variable names as they can clash with the DSL methods of the router.
+Since this is a regular Ruby source file you can use all of Ruby's features to help you define your routes.
```

Comment:
What about?
```suggestion
Since this is a regular Ruby source file, you can use all of Ruby's features (like conditionals and loops) to help you define your routes.
```